docker/uwsgi.yml

```diff
@@ -45,7 +45,3 @@ uwsgi:
   ignore-sigpipe: true
   ignore-write-errors: true
   disable-write-exception: true
-
```

Comment:
This monitoring is not related to Prometheus. It's for uWSGI, but I'm not completely sure what for. [Some statistics](https://dev.azure.com/City-of-Helsinki/tilavarauspalvelu/_git/tilavarauspalvelu-pipelines?path=/devops/tilavarauspalvelu-core-service.yml)?

Reply:
I think we shouldn't remove it until we know what it does.

Reply:
According to the [PR it was added in](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/721), it should only be used for Prometheus-related stuffs 🤔